game/opengl_text.cc

```diff
@@ -212,11 +135,13 @@ SvgTxtThemeSimple::SvgTxtThemeSimple(fs::path const& themeFile, float factor) :
 	parseTheme(themeFile, m_text, tmp, tmp, tmp, tmp, a);
 }
 
-void SvgTxtThemeSimple::render(std::string _text) {
-	if (!m_opengl_text.get() || m_cache_text != _text) {
-		m_cache_text = _text;
-		m_text.text = _text;
-		m_opengl_text = std::make_unique<OpenGLText>(m_text, m_factor);
+void SvgTxtThemeSimple::render(std::string const& text) {
+	if (!m_opengl_text.get() || m_cache_text != text) {
+		m_cache_text = text;
+
+		static TextRenderer renderer;
```

Comment:
Would it make sense to make this a member field within opengl_text.hh?

Reply:
I think that this would actually have no benefit.